<launch>
  <node name="turning_control" pkg="vehicle_control" type="DualSensor2DirController.py"></node>
  <node name="turning_pid" pkg="pid" type="controller" >
    <param name="Kp" value="5.0" />
    <param name="Ki" value="0.0" />
    <param name="Kd" value="0.1" />
    <param name="upper_limit" value="0.78539816339" />
    <param name="lower_limit" value="-0.78539816339" />
    <param name="windup_limit" value="1" />
    <param name="topic_from_controller" value="steering/control_effort" />
    <param name="topic_from_plant" value="steering/state" />
    <param name="setpoint_topic" value="steering/setpoint" />
    <param name="max_loop_frequency" value="100.0" />
    <param name="min_loop_frequency" value="100.0" />
  </node>
  <node name="throttle_pid" pkg="pid" type="controller" >
    <param name="Kp" value="5.0" />
    <param name="Ki" value="0.0" />
    <param name="Kd" value="0.1" />
    <param name="upper_limit" value="0.78539816339" />
    <param name="lower_limit" value="-0.78539816339" />
    <param name="windup_limit" value="1" />
    <param name="topic_from_controller" value="throttle/control_effort" />
    <param name="topic_from_plant" value="throttle/state" />
    <param name="setpoint_topic" value="throttle/setpoint" />
    <param name="max_loop_frequency" value="100.0" />
    <param name="min_loop_frequency" value="100.0" />
  </node>
  <node name="turning_broadcaster" pkg="vehicle_control" type="Controller2DirMotor.py"></node>
</launch>
